source/contribute.html.markdown

```diff
@@ -26,8 +26,4 @@ If you just want to report a bug, that's easily done in our [github project issu
 
 ## Where To Ask For Help
 
-Need some help getting started? Feel free to ask on the [mailing list](/mailing-lists.html) or find us on one of the following IRC channels on irc.freenode.net:
-
-* \#cloudstack - General Apache CloudStack conversation and end user support 
-* \#cloudstack-dev - Development discussions 
-* \#cloudstack-meeting - Weekly and ad-hoc meeting room for the Apache CloudStack community
+Need some help getting started? Feel free to ask on the [mailing list](/mailing-lists.html).
```

Comment:
should we mention github/slack here?

Reply:
Thanks for the review @DaanHoogland. I am not 100% sure about adding Slack, particularly I like to hang on Slack sometimes to reach developers. But the discussion on the mailing list ([ML reference](https://lists.apache.org/thread.html/296def97c549e0b75bc14d0382ae412245cef6088e576acbcb007cc8%40%3Cdev.cloudstack.apache.org%3E)) did not reach a consensus regarding Slack.

The same question I had here: https://github.com/apache/cloudstack-documentation/pull/204

Reply:
Yes, makes sense @GabrielBrascher , maybe a general invitation to seek out new communication means and make them known.